buildpacks/jvm/src/bin/heroku_metrics_agent_setup.rs

```diff
@@ -50,7 +50,7 @@ fn output_from_env(env: &Env) -> HashMap<ExecDProgramOutputKey, String> {
 }
 
 #[cfg(test)]
-mod tests {
+mod test {
```

Comment:
`tests` seems to be the norm, rather than the other way around? eg:
https://doc.rust-lang.org/rust-by-example/testing/unit_testing.html

Reply:
That's true. Honestly, I based it off of what is used most and it looks like the "wrong" name was more prevalent. I will change this to `tests` to fit better into the ecosystem. Something felt off, now I know what that was. :) Thanks!

Reply:
Fixed in [c4798de](https://github.com/heroku/buildpacks-jvm/pull/483/commits/c4798dedac347354ce4cb261601555744c8c7d4b)